lte/gateway/c/core/oai/tasks/amf/amf_app_handler.cpp

```diff
@@ -51,6 +52,7 @@ void amf_ue_context_update_coll_keys(
     const amf_ue_ngap_id_t amf_ue_ngap_id, const imsi64_t imsi,
     const teid_t amf_teid_n11, const guti_m5_t* const guti_p) {
   hashtable_rc_t h_rc                 = HASH_TABLE_OK;
+  map_rc_t m_rc                       = MAP_OK;
   hash_table_ts_t* amf_state_ue_id_ht = get_amf_ue_state();
```

Comment:
can we add a `TODO` here to note that we want to migrate this hash_table as well?

Reply:
This comment is addressed.